package/lib/components/ConnectPortalDialog.tsx

```diff
@@ -36,20 +34,30 @@ export const ConnectPortalDialog: React.FC<ConnectPortalDialogProps> = (
 
   return (
     <PlainDialog isOpen={props.isOpen} onClose={handleClose}>
+      <svg
```

Comment:
Maybe we want a parameter to control showing this header logo or not

Reply:
Added logo customization config